Yoast/wordpress-seo
# Make the configuration wizard back link look like a button.

## Summary

This PR can be summarized in the following changelog entry:

* Make the configuration wizard "go back" link look like a button

## Relevant technical choices:

* made on the PHP side using just CSS

## Test instructions

This PR can be tested by following these steps:

* open the configuration wizard, check the styling of the button on hover, focus, active

Fixes https://github.com/Yoast/yoast-components/issues/246
